File: URI-C++/1017.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
	
	int tempo, velocidade;
	float gasto;
	
	cin >> tempo;
	cin >> velocidade;
	
	gasto = (tempo * velocidade) / 12.0;
	
	cout <<  fixed << setprecision(3) << gasto << endl;
	
	return 0;
}

File: Go/1017.go
package main

import (
	"fmt"
)

func main() {
	var tempo, velocidade int 

	fmt.Scanf("%d", &tempo)
	fmt.Scanf("%d", &velocidade)

	gasto := (float64(tempo) * float64(velocidade)) / 12.0
	fmt.Printf("%.3f\n", gasto)
}
